### PR TITLE
Restyle untrusted SSO authorization form

### PR DESCRIPTION
**Pull request by RoyEJohnson**
View comes from doorkeeper gem, but styling is added locally, scoped to `.authorizations.new`

### CODE REVIEW

**Comment by RoyEJohnson** [comment]
image

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -61,3 +61,30 @@
     padding: 0px 10px;
   }
 }
+
+.authorizations.new {
+    #application-body {
```
Comment (nathanstitt) [inline app/assets/stylesheets/application.scss]: You might consider adding the `ox-card` style to the view, that way it'll match the other "card type" views and can use the built-in styles https://github.com/openstax/accounts/blob/master/app/assets/stylesheets/controls/_card.scss

Will probably still have to style the buttons a bit though

Comment (RoyEJohnson) [reply app/assets/stylesheets/application.scss]: The view seems to be owned by the doorkeeper gem, as you suggested, so unless I make a local view (and I suspect that's beyond my skills), I can't add a class. If `ox-card` were defined as a placeholder type, though, I could use that.

Comment (nathanstitt) [reply app/assets/stylesheets/application.scss]: ah gotcha.  Yep this makes sense then

**Pull request merged**